static/js/util/api.js

```diff
@@ -1,4 +1,4 @@
-// @flow
+// @flow weak
```

Comment:
(i.e. I think it should be removed before merging, but I'm not quite sure how to handle assigning the value we've type-annotated as `string` to be `null below)